etc/databases/world/updates/updates.sql

```diff
@@ -1474,5 +1474,49 @@ begin not atomic
 		insert into applied_updates values ('260320231');
 	end if;
 
+    -- 27/03/2023 1
+	if(select count(*) from applied_updates where id = '270320231') = 0 then
+		-- Placeholder display_id for Bridge Workers, correct levels & faction
+		update creature_template set display_id1 = 310, faction = 11 where entry in(648, 649, 650, 651, 652, 653);
+		update creature_template set level_min = 6, level_max = 6 where entry in(650, 651, 652, 653, 652, 653);
+		update creature_template set level_min = 10, level_max = 10 where entry = 649; -- Bridge Worker Dmitri only
+		update creature_template set level_min = 5, level_max = 5 where entry = 648; -- Bridge Worker Trent only
```

Comment:
Where are the levels taken from? Also, I think we should spawn the remaining 2 workers even if they don't appear here:
![image](https://user-images.githubusercontent.com/8654362/227925507-62a9a6b3-fbb5-4625-b7a8-b025b1e274a5.png)
probably they were spawned in front of the 3 that are next to the campfire, to the other side of it.

Reply:
Geo found the levels listed on a website and posted them within #1067 
I left the others unspawned because we don't know there exact position of them yet but I could spawn them on the other side of the campfire if you want me to.

Reply:
Go for it. :)